README.md

```diff
@@ -1,70 +1,202 @@
-# Getting Started with Create React App
+<div align="center">
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+  <h3><b>Hello Rails React Frontend App</b></h3>
 
-## Available Scripts
+  Link for Rails React Backend: https://github.com/dheerajsachdeva/hello_rails_backend
 
-In the project directory, you can run:
+</div>
 
-### `npm start`
+<!-- TABLE OF CONTENTS -->
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+# 📗 Table of Contents
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
```

Comment:
- [ ] It would be nice if you fix your Deployment link because it is broken 😨 Kindly fix it or remove it if it doesn't have any section to make your readme descriptive and professional 👍

Reply:
@HAMMAS-SALEEM Done, Thanks